README

```diff
@@ -0,0 +1,39 @@
+<div align="center">
```

Comment:
Something is weird here https://github.com/snapshot-labs/discord-bot/tree/setup-discord-bot
I see plain html 🙈 not sure why
![Untitled 3](https://user-images.githubusercontent.com/15967809/215857333-5a750cdf-da23-4a79-ab8a-70bd26d73612.png)